usaspending_api/etl/es_transaction_template.json

```diff
@@ -47,6 +47,11 @@
         "format": "yyyy-MM-dd HH:mm:ss||yyyy-MM-dd||epoch_millis",
         "index": false
       },
+      "etl_update_date": {
+        "type": "date",
+        "format": "yyyy-MM-dd HH:mm:ss||yyyy-MM-dd||epoch_millis",
+        "index": false
```

Comment:
why `"index": false`? 

I haven't see yet where this is used, but if it's used at all in ES queries (i.e. to find records for delete/upsert), it should be indexed.

Reply:
Why does it need to be indexed? The current field `update_date` is not indexed